client/css/admin.css

```diff
@@ -18,8 +18,20 @@ table th, table td {
     padding: 12px 4px;
 }
 table td:last-child {
-    text-align: center;
     width: 50px;
+    text-align: center;
+}
+#applicants table td:last-child {
+    width: 140px;
```

Comment:
The used of `px` in principle triggers me. But it 200% doesn't matter. Just my 2 cents.